daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/MetadataWalker.scala

```diff
@@ -0,0 +1,76 @@
+package org.apache.daffodil.runtime1.processors
```

Comment:
Comment from prior PR:
https://github.com/apache/daffodil/pull/1092/files#r1360834018

Asked if this should be in some specific different package like perhaps "dfdlx.japi" or japix ?